src/test/java/io/stargate/sgv2/jsonapi/exception/playing/ErrorTestData.java

```diff
@@ -6,9 +6,9 @@
  */
 public class ErrorTestData {
 
-  public static final String TEST_ERROR_CONFIG_FILE = "test_errors.yaml";
+  public final String TEST_ERROR_CONFIG_FILE = "test_errors.yaml";
```

Comment:
Why not `static` as constants typically are?

Reply:
from memory it was because when they are static, and i create an instance of the class then access the static the IDE was giving a linting warning about accessing a static field from an instance. 

checked and the IDE gives this warning if I make it instance: Static member 'io.stargate.sgv2.jsonapi.exception.playing.ErrorTestData.TEST_ERROR_CONFIG_FILE' accessed via instance reference 

We should come up with a common pattern for the test data classes now that we have a few and they are making the tests a little easier to read.

Reply:
oh and prob some ideas on the linting we want to follow

Reply:
Ah yes. True wrt access via instance. Not a big deal either way.